<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1076</width>
     <height>677</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>371</width>
      <height>700</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>307</width>
       <height>267</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Моделирование роста дерева</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="generate_growth">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Смоделировать</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="previous">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Назад</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="play">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Play</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stop">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="reverse">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Reverse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="next">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Вперед</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Скорость:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="speed">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="end_modeling">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Закончить моделирование</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>160</width>
       <height>184</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="AddButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Добавить новую
камеру</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AddBaseCamera">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Добавить базовую
 камеру</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DeleteButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Удалить текущую
камеру</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="CameraBox"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>156</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Цвет фона:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="font_color">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>194</width>
       <height>166</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="LoadButton">
        <property name="text">
         <string>Загрузить сохраненную
модель</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="generate_new">
        <property name="text">
         <string>Создать новую
модель</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_model">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Удалить модель</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="save_new">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Сохранить модель</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="Drawer_Scene" name="scene" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Drawer_Scene</class>
   <extends>QWidget</extends>
   <header location="global">drawer_scene.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
